### PR TITLE
Authentication OK

### PR DESCRIPTION
**Pull request by LauraCourquin**

### CODE REVIEW

File: frontend/src/components/RegisterOne/index.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Link } from "react-router-dom";
+import SRegisterOne from "./style";
+
```
Comment (ghost) [inline frontend/src/components/RegisterOne/index.jsx:3]: Blank space

File: frontend/src/components/SignupArtist/index.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import SSignupArtist from "./style";
+
+export default function SignupArtist() {
+  const [formData, setFormData] = useState({
+    bandName: "",
+    lastname: "",
+    firstname: "",
+    email: "",
+    password: "",
+    secuNum: "",
```
Comment (ghost) [inline frontend/src/components/SignupArtist/index.jsx]: No secuNum in us project

**Review by ghost — COMMENTED** [review]
some fix

**Review by LauraCourquin — COMMENTED** [review]
Update

**Review by ghost — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me

File: backend/src/router.js [hunk 3]
```diff
@@ -29,4 +28,16 @@ router.post("/auth/signup", AuthController.signup);
...
+router.get("/user/:id", UserController.read);
+router.put("/user/:id", UserController.edit);
+router.post("/user", UserController.add);
+router.delete("/user/:id", UserController.delete);
+
+router.get("/artist", ArtistController.browse);
+router.get("/artist/:id", ArtistController.read);
+router.put("/artist/:id", ArtistController.edit);
+router.post("/artist", ArtistController.add);
+router.delete("/artist/:id", ArtistController.delete);
```
Comment (LoicBrassart) [inline backend/src/router.js:41]: Nothing to do about authentication

File: frontend/src/App.jsx [hunk 4]
```diff
@@ -15,7 +16,8 @@ function App() {
         <Route path="/profile" element={<Profile />} />
         <Route path="/profile/:profileId" element={<Profile />} />
         <Route path="/register" element={<Register />} />
-        <Route path="/register/artist" element={<RegisterArtist />} />
+        <Route path="/register/artist" element={<SignupArtist />} />  
+        <Route path="/register/artist/validation" element={<RegisterArtist />} />
```
Comment (LoicBrassart) [inline frontend/src/App.jsx:20]: No auth

File: frontend/src/components/SignupArtist/index.jsx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+  const [checkTwo, setCheckTwo] = useState(false);
+
+  const handleChange = () => {
+    setCheckOne(!checkOne);
+    setCheckTwo(false);
+  };
+  const handleChangeBis = () => {
+    setCheckTwo(!checkTwo);
+    setCheckOne(false);
+  };
```
Comment (LoicBrassart) [inline frontend/src/components/SignupArtist/index.jsx:30]: You already renamed these to `hChange` IIRC

**Review by LoicBrassart — APPROVED** [review]
Didn't read most of it :( 
I'll let it pass, I hope it'll go well :shrug:

**Pull request merged**